app/models.py

```diff
@@ -109,8 +131,8 @@ def serialize(self):
             'id': self.supplier_id,
             'name': self.name,
             'description': self.description,
-            # 'dunsNumber': self.duns_number,
-            # 'eSourcingId': self.esourcing_id,
+            'dunsNumber': self.duns_number,
```

Comment:
Apparently we would have to pay money to serve the Duns numbers (although while the API is private this probably isn't an issue).  We should talk with Cath & @minglis about whether these will ever be used for anything.  If not then maybe we should drop them altogether.